modules/media/ajax/FileUpload.php

```diff
@@ -42,15 +42,14 @@ function editFile()
 
     // Process posted data
     $idMediaFile = $_POST['idMediaFile'];
-    $dateTaken   = isset($_POST['dateTaken']) ? $_POST['dateTaken'] : null;
-    $comments    = isset($_POST['comments']) ? $_POST['comments'] : null;
-    $hideFile    = $_POST['hideFile'];
 
-    $updateValues = [
-                     'date_taken' => $dateTaken,
-                     'comments'   => $comments,
-                     'hide_file'  => $hideFile,
-                    ];
+    $updateValues['hide_file'] = $_POST['hideFile'];
+    if (!empty($_POST['dateTaken']) && $_POST['dateTaken'] != 'null') {
+        $updateValues['date_taken'] = $_POST['dateTaken'];
+    }
```

Comment:
should this have an else statement to explicitly set it to null? The way this is written if dateTaken='null' is POSTed, it will leave the old value unchanged.

(same for the line below)